src/Aspire.Cli/Commands/RunCommand.cs

```diff
@@ -218,7 +218,7 @@ protected override async Task<int> ExecuteAsync(ParseResult parseResult, Cancell
                 cancellationToken);
 
             // Wait for the backchannel to be established.
-            var backchannel = await InteractionService.ShowStatusAsync(InteractionServiceStrings.BuildingAppHost, async () => { return await backchannelCompletitionSource.Task.WaitAsync(cancellationToken); });
+            var backchannel = await InteractionService.ShowStatusAsync(isExtensionHost ? InteractionServiceStrings.BuildingAppHost : RunCommandStrings.ConnectingToAppHost, async () => { return await backchannelCompletitionSource.Task.WaitAsync(cancellationToken); });
```

Comment:
Is this logic correct?

Reply:
It is, since the app host will be built in the aspire extension